gulp/tasks/styles.js

```diff
@@ -259,6 +271,9 @@ function stylesNemoIE() {
  * @returns {PassThrough} A source stream.
  */
 function stylesOAH() {
+  const browserList = environment.getSupportedBrowserList( 'stylesModern' );
```

Comment:
This is unused in this function.

Reply:
Whoops, forgot to use it farther down. Fixed.